.github/workflows/R-CMD-check.yaml

```diff
@@ -27,92 +23,74 @@ jobs:
       matrix:
         config:
           - {os: macOS-latest,   r: 'release'}
+
           - {os: windows-latest, r: 'release'}
-          - {os: windows-latest, r: '3.6', rspm: "https://packagemanager.rstudio.com/cran/latest"}
-          - {os: ubuntu-18.04,   r: 'devel', rspm: "https://packagemanager.rstudio.com/cran/__linux__/bionic/latest", http-user-agent: "R/4.1.0 (ubuntu-18.04) R (4.1.0 x86_64-pc-linux-gnu x86_64 linux-gnu) on GitHub Actions" }
```

Comment:
GitHub Actions is no longer supporting Ubuntu 18.04, so we switched to 20.04.

Note that r-lib/actions actually just switched to `ubuntu-latest` as the default, but I think for cpp11 it is worth trying to continue to test on "older" Ubuntu versions like 20.04 https://github.com/r-lib/actions/pull/612